checkov/secrets/scan_git_history.py

```diff
@@ -200,3 +214,29 @@ def _run_scan_one_commit(commit: Commit) -> Tuple[List[RawStore], int]:
                                     rename_from=rename_from, rename_to=rename_to))
             scanned_file_count += 1
         return results, scanned_file_count
+
+    @time_it
+    def _get_first_commit(self) -> List[Commit]:
```

Comment:
@marynaKK I'm surprised this returns a list and not just a commit... The first one

Reply:
you are right. I need to adjust the test to return an empty commit.